android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/TrackViewHolder.java

```diff
@@ -45,9 +51,16 @@ public void bindTrack(Track track) {
         this.track = track;
 
         int trackColor = Color.parseColor(track.getColor());
+        int sessions = track.getSessions().size();
         String trackName = Utils.checkStringEmpty(track.getName());
 
         trackTitle.setText(trackName);
+        noOfSessions.getBackground().setColorFilter(trackColor, PorterDuff.Mode.SRC_ATOP);
+        if (sessions != 1)
+            noOfSessions.setText(String.format("%d sessions", sessions));
```

Comment:
I think it'd be just enough to write `session(s)`

Also, use strings.xml to store this